recipe/meta.yaml

```diff
@@ -7,55 +7,57 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/meson_python-{{ version }}.tar.gz
-  sha256: 3d5b3e581d70a58a97b9c116a75e97a76cc4b4c7e75fa0658fc8392371e2f2c2
+  sha256: 63b3170001425c42fa4cfedadb9051cbd28925ff8eed7c40d36ba0099e3c7618
 
 build:
   skip: True # [py<37]
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation 
   number: 0
 
 requirements:
-  build:
-    - patch     # [not win]
-    - m2-patch  # [win]
   host:
-    - meson >=0.63.3
-    - ninja
+    - meson 1.0.1
+    - ninja-base 1.10.2
```

Comment:
I struggle to see why ninja is needed. Where is it being used?

Reply:
You are right. It's not needed. I've removed it from host and run